torch/_dynamo/variables/misc.py

```diff
@@ -843,14 +861,17 @@ def call_function(
                 unimplemented(
                     f"{self.value.__name__} with non-const args: {args} {kwargs}"
                 )
-        else:  # We are dealing with a callable.
-            func = get_np_to_tnp_map().get(self.value)
-            if func is None:
-                unimplemented(
-                    f"Can't find numpy function {self.value} in torch._numpy. "
-                    " Please file an issue to request support for this function."
-                )
-
+        if self.can_constant_fold_through(func) and (
```

Comment:
why is this block of code now up here?

Reply:
I will restore it so the diff is cleaned up.